dozer-types/src/types/mod.rs

```diff
@@ -204,6 +206,64 @@ impl Record {
         }
         res_buffer
     }
+
+    pub fn get_values_hash(&self) -> u64 {
```

Comment:
`Hash` is derived but not used? And I feel there's a reason that we hash field indexes and field types but I can't figure it out.

Reply:
Not using the derived Hash to compare the content of the record, just values, without version or other stuff.

Reply:
it's based on a previous implementation;
My assumption was same values submitted in different order can generate the same hash, that's why we include more informations?

but need a comment from @snork-alt on this

Reply:
Then let's not derive it?

Reply:
oh right, if we don't need, we can remove from derive